ext/ITensorNetworksEinExprsExt/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,52 @@
+module ITensorNetworksEinExprsExt
+
+using ITensors: Index, ITensor, @Algorithm_str, inds, noncommoninds
+using ITensorNetworks:
+  ITensorNetworks, ITensorNetwork, vertextype, vertex_data, contraction_sequence
+using EinExprs: EinExprs, EinExpr, einexpr, SizedEinExpr
+
+function to_einexpr(ts::Vector{ITensor})
+  IndexType = Any
+
+  tensor_exprs = EinExpr{IndexType}[]
+  inds_dims = Dict{IndexType,Int}()
+
+  for tensor_v in ts
+    inds_v = collect(inds(tensor_v))
+    push!(tensor_exprs, EinExpr{IndexType}(; head=inds_v))
+    merge!(inds_dims, Dict(inds_v .=> size(tensor_v)))
+  end
+
+  externalinds_tn = reduce(noncommoninds, ts)
+  return SizedEinExpr(sum(tensor_exprs; skip=externalinds_tn), inds_dims)
+end
+
+function tensor_inds_to_vertex(ts::Vector{ITensor})
+  IndexType = Any
+
+  mapping = Dict{Set{IndexType},VertexType}()
```

Comment:
Looks like `VertexType` is also not defined.

Reply:
Ah, damn.